src/sql/base/browser/ui/table/plugins/headerFilter.plugin.ts

```diff
@@ -204,7 +220,7 @@ export class HeaderFilter<T extends Slick.SlickData> {
 		}
 
 		if (!this.$menu) {
-			this.$menu = jQuery('<div class="slick-header-menu">').appendTo(document.body);
+			this.$menu = jQuery('<div class="slick-header-menu">').appendTo(container);
```

Comment:
are we using `this.$menu` and the `contextViewService` both now? Can we get rid of the jquery calls and just use the `contextViewService` for all actions?

Reply:
$in the '$menu' is just part of the variable name, nothing to do with JQuery here.

Reply:
contextView service and jquery has no conflict, right?